.github/workflows/canary-gpu-cuda-113.yml

```diff
@@ -38,13 +38,6 @@ jobs:
         with:
           java-version: 11
           distribution: corretto
-      - name: Test MXNet
-        working-directory: canary
-        run: |
-          DJL_ENGINE=mxnet-native-auto ./gradlew clean run
-          rm -rf ~/.djl.ai
-          DJL_ENGINE=mxnet-native-mkl ./gradlew clean run
```

Comment:
We just want to make sure CPU build can run on GPU machine.

Reply:
I'm a bit confused on cuda 11.2 for mxnet - it seems like we publish a library to s3 for 11.2 support in S3, and have it documented as well in our code base https://github.com/deepjavalibrary/djl/blob/master/engines/mxnet/mxnet-engine/README.md#linux-gpu

Reply:
MXNet only support CUDA 11.2, we used to have a bug trying to load GPU version on CU113, so we add this test here to make sure we fallback to CPU on CU113